monai/data/utils.py

```
@@ -661,67 +668,72 @@ def partition_dataset_classes(
             if False, will add extra indices to make the data evenly divisible across partitions.
         even_divisible: if True, guarantee every partition has same length.
 
-    Examples:
-        data: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14]
-        classes: [2, 0, 2, 1, 3, 2, 2, 0, 2, 0, 3, 3, 1, 3]
-        shuffle: False, ratios: [2, 1]
-        output: [[2, 8, 4, 1, 3, 6, 5, 11, 12], [10, 13, 7, 9, 14]]
+    Examples::
+
+        >>> data = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14]
+        >>> classes = [2, 0, 2, 1, 3, 2, 2, 0, 2, 0, 3, 3, 1, 3]
+        >>> partition_dataset_classes(data, classes, shuffle=False, ratios=[2, 1])
+        [[2, 8, 4, 1, 3, 6, 5, 11, 12], [10, 13, 7, 9, 14]]
 
     """
-    data_len = len(data)
     datasets = list()
+    if not classes:
+        return []
```

I think we should raise an exception directly if no classes provided.
Thanks.